OWNERS.md

```diff
@@ -50,7 +50,6 @@ They're not part of the TSC voting process, but appreciated for their contributi
 * Carlos ([@nzlosh](https://github.com/nzlosh)) - Chatops, Errbot, Community, Discussions, StackStorm Exchange.
 * Hiroyasu Ohyama ([@userlocalhost](https://github.com/userlocalhost)) - Orquesta, Workflows, st2 Japan Community. [Case Study](https://stackstorm.com/case-study-dmm/).
 * Jon Middleton ([@jjm](https://github.com/jjm)) - StackStorm Exchange, Core, Discussions.
```

Comment:
@StackStorm/tsc FYI I just saw @jjm in https://github.com/StackStorm-Exchange/stackstorm-orion/pull/23, he's still interested in participating more in st2 community and there is a chance st2 will be used in the new workplace potentially leading to more involvement in the StackStorm development.